pkg/vscode/v1/bin/merge-vscode-proto3-settings

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+if ! hash jq 2>/dev/null; then
+    echo "no jq binary is installed in the executable path, merging vscode-protoc3 extension settings is ignored"
+    exit 0
+fi
+
+if [[ "$#" -lt 3 ]]; then
+	echo "this script needs at least 3 arguments"
+    exit 1
+fi
+
+settings_file="${1}"
+protoc_path="${2}"
+arg_files="${@:3}"
+
+out=$(cat ${settings_file} | jq --arg path "${protoc_path}" '. * {"protoc":{"path":$path, "compile_on_save":false, "options":["--proto_path=."]}}')
+
+for option in $(cat ${arg_files})
+do
+    out=$(echo "${out}" | jq --arg option "${option}" '.protoc.options |= . + [$option]')
```

Comment:
Hrm, will this approach duplicate settings when `go.mod` changes, for example?

Reply:
No it won't (and this have already been tested) as `*` jq's operator merges the json with existing one and the previous `options` array is overwritten with just `"options":["--proto_path=."]` .